Solutions/MCBSTM32F400/TinyCLR/scatterfile_tinyclr_mdk.xml

```diff
@@ -50,7 +50,7 @@
 
     <!-- DAT region goes into external NOR Flash -->
     <Set Name="Data_BaseAddress"        Value="%Config_BaseAddress% + %Config_Size%"/>
-    <Set Name="Data_Size"               Value="0x00020000"/><!-- 128KB -->
+    <Set Name="Data_Size"               Value="0x00030000"/><!-- 128KB -->

```

Comment:
Sorry for that. I am out town until Monday. Will do it when I am back.

Sent from my Windows Phone
________________________________
From: Steve Maillet<mailto:notifications@github.com>
Sent: ‎30/‎5/‎2015 7:27
To: NETMF/netmf-interpreter<mailto:netmf-interpreter@noreply.github.com>
Cc: Jennifer Cheng (Adecco)<mailto:a-jennch@microsoft.com>
Subject: Re: [netmf-interpreter] Fixed the build break for MCBStm32F400, the LR_dat assigned size is n… (#134)


In Solutions/MCBSTM32F400/TinyCLR/scatterfile_tinyclr_mdk.xml<https://github.com/NETMF/netmf-interpreter/pull/134#discussion_r31373158>:

> @@ -50,7 +50,7 @@
>
>      <!-- DAT region goes into external NOR Flash -->
>      <Set Name="Data_BaseAddress"        Value="%Config_BaseAddress% + %Config_Size%"/>
> -    <Set Name="Data_Size"               Value="0x00020000"/><!-- 128KB -->
> +    <Set Name="Data_Size"               Value="0x00030000"/><!-- 128KB -->


0x00030000 != 128KB, please be sure comments are correctly updated to reflect changes!

—
Reply to this email directly or view it on GitHub<https://github.com/NETMF/netmf-interpreter/pull/134/files#r31373158>.